Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/GraphSerialization.cpp

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/IO/GenericStreams.h>
+#include <AzCore/Serialization/Json/JsonSerialization.h>
+#include <AzCore/Serialization/Json/JsonSerializationResult.h>
+#include <AzCore/Serialization/Json/JsonUtils.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/Utils.h>
+#include <AzCore/Utils/Utils.h>
+#include <AzCore/std/string/string_view.h>
+#include <ScriptCanvas/Core/Graph.h>
+
+#include <ScriptCanvas/Core/GraphSerialization.h>
+
+namespace GraphSerializationCpp
+{
+    void AppendTabs(AZStd::string& result, size_t depth)
+    {
+        for (size_t i = 0; i < depth; ++i)
+        {
+            result += "\t";
+        }
+    }
```

Comment:
You could use AZStd:fill_n here
```suggestion
        AZStd::fill(AZStd::back_inserter(result), depth, '\t');
```